test/controlplane/Makefile

```diff
@@ -7,3 +7,15 @@ test:
 
 update-golden:
 	go test . -test.v -update
+
+pre-pull-kind-images:
+	bash ./k8s_versions.sh --pre-pull-images
+
+update-k8s-versions:
+	# Delete all auto-generated input test files
+	find . -type d -regextype posix-extended -regex ".*/v[0-9]{1}\.[0-9]{2}" | xargs rm -r || true
```

Comment:
any reason to remove files here? is it mainly to clean up old version?

I am thinking about only overwrite existing file but not remove, we can only remove if the kube version is not longer mentioned in k8s_versions.txt.

Reply:
Great idea.